corehq/blobs/models.py

```diff
@@ -4,3 +4,17 @@
 class BlobMigrationState(models.Model):
     slug = models.CharField(max_length=20, unique=True)
     timestamp = models.DateTimeField(auto_now=True)
+
+
+class BlobExpiration(models.Model):
+    '''
+    This models records when temporary blobs should be deleted
+    '''
+    bucket = models.CharField(max_length=255)
+    identifier = models.CharField(max_length=255)
+    expires_on = models.DateTimeField(db_index=True)
+    created_on = models.DateTimeField(auto_now=True)
+    length = models.IntegerField()
```

Comment:
this was discussed in the scale meeting as a suggestion. it's way to easily keep track of how much temp storage we have (it's been an issue on icds recently)